relayer/options.go

```diff
@@ -38,3 +38,15 @@ func ImagePull(pull bool) RelayerOption {
 }
 
 func (opt RelayerOptionImagePull) relayerOption() {}
+
+type RelayerOptionStartupFlags struct {
```

Comment:
Maybe `RelayerOptionExtraStartFlags`? Since the method is `Start`, and these are additional flags, not overriding the existing ones.

Reply:
done